File: ui_automation_demoqa/src/main/java/pages/HomePage.java
package pages;

import com.microsoft.playwright.Page;

public class HomePage {
    private Page page;

    public HomePage(Page page) {
        this.page = page;
    }

    public void clickOnElementsCard() {
        page.locator("//h5[text()='Elements']").click();
    }
}

File: ui_automation_demoqa_python/src/pages/home_page.py
from selenium.webdriver.common.by import By
from selenium.webdriver.support.wait import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC


class HomePage:
    def __init__(self, driver):
        self.driver = driver
        self.wait = WebDriverWait(driver, 20)
        
        # Locators
        self.elements_card = (By.XPATH, "//h5[text()='Elements']")
    
    def click_on_elements_card(self):
        """Click on Elements card"""
        element = self.wait.until(EC.element_to_be_clickable(self.elements_card))
        element.click()